src/System.Data.SqlClient/tests/ManualTests/SQL/SqlNotificationTest/SqlNotificationTest.cs

```diff
@@ -37,9 +37,13 @@ public void Dispose()
             Cleanup();
         }
 
+        public static bool IsServiceBrokerEnabled() => DataTestUtility.IsServiceBrokerEnabled();
```

Comment:
You could get rid of these two statics and use something like

`[ConditionalFact(nameof(DataTestUtility), nameof(DataTestUtility.IsServiceBrokerEnabled), nameof(DataTestUtility.AreConnectionStringsSetup))] `

This comment applies to all the new statics that have been added in this PR, that passthrough to DataTestUtility.*() method. 
E.g. 
https://github.com/dotnet/corefx/blob/a10890f4ffe0fadf090c922578ba0e606ebdd16c/src/System.Diagnostics.PerformanceCounter/tests/PerformanceCounterTests.cs#L15

Reply:
Done.